include/ts/apidefs.h.in

```diff
@@ -655,6 +655,15 @@ char const TS_VERSION_STRING[] = "@TS_VERSION_STRING@";
     TS_TXN_INFO_LAST_ENTRY,
   };
 
+  enum TSHttpSsnInfoKey {
+    TS_SSN_INFO_NONE = -1,
+    TS_SSN_INFO_TRANSACTION_COUNT,
+    TS_SSN_INFO_RECEIVED_FRAME_COUNT,
+  };
+
+#define TS_SSN_INFO_RECEIVED_FRAME_COUNT_H2_UNKNOWN 999
+#define TS_SSN_INFO_RECEIVED_FRAME_COUNT_H3_UNKNOWN 0x21
```

Comment:
Why is this 0x21 when the index enum value is 0x0e?

Reply:
Because 0x0e is just an unassigned number (it could be used later for something). 0x21 is reserved for exercising and will never be used.
https://datatracker.ietf.org/doc/html/rfc9114#name-reserved-frame-types